DOC-2135: bug fix documentation entry for TINY-10078 in the 6.6.1-release-notes.adoc (#2835)
tinymce/tinymce-docs: modules/ROOT/pages/6.6.1-release-notes.adoc

{productname} 6.6.1 also includes the following bug fixes:

=== On Safari and Firefox, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component
//#TINY-10078

On each content update for iframe dialog components with `streamContent: true` set, the iframe is expected to automatically scroll to the bottom if the pre-update scrolled position is already at the end of the scrollable area. It is also expected, however, that the end-user can freely scroll through the iframe as content updates occur.

When Chrome is the host browser, scroll positions are natively maintained after `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.

> As a consequence, once the user scrolls up in the iframe as content updates occur, the programmatic scrolling to the bottom no longer occurs. Instead, the native behavior takes over and keeps the scroll at the top, even as content updates continue.

Reviewer: ```suggestion
As a consequence, once the user scrolls up in the iframe as content updates occur and the programmatic scrolling to the bottom no longer occurs, the native behavior takes over. On Safari and Firefox, this results in the scroll being kept at the top even as the user attempts to scroll, which is both against expected behavior and poor UX.
```

> This makes the {productname} dialog feel stuck when Safari or Firefox are the host browsers, which is both against expected behavior and poor UX.

Reviewer: ```suggestion
```

> === On Safari and Firefox browsers, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component

Author: ```suggestion
=== On Safari and Firefox, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component
```

//#TINY-10078

On each content update for iframe dialog components with `streamContent: true` set, the iframe is expected to automatically scroll to the bottom if the pre-update scrolled position is already at the end of the scrollable area. It is also expected, however, that the end-user can freely scroll through the iframe as content updates occur.

> When Chrome is the host browser, scroll positions are natively maintained after `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.

Reviewer 2: ```suggestion
On Chrome, scroll positions are natively maintained after updates, but this is not true when on either Safari or Firefox. Both these browsers reset the scroll positions to the top.
```

Reviewer: It is important to keep the fact that the content updates are made by `document.write()`, since the scrolling behavior is different when using other methods. @brian-forte Feel free to combine Ben's and my changes to your desire.
```suggestion
When Chrome is the host browser, scroll positions are natively maintained after content updates via `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.
```

As a consequence, once the user scrolls up in the iframe as content updates occur, the programmatic scrolling to the bottom no longer occurs. Instead, the native behavior takes over and keeps the scroll at the top, even as content updates continue.

This makes the {productname} dialog feel stuck when Safari or Firefox are the host browsers, which is both against expected behavior and poor UX.

> As of {productname} 6.6.1, the extant `scrollTop` value before any content updates happen is stored. After a content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically restore the previous scroll position using the stored `scrollTop` value.

Reviewer 2: ```suggestion
As of {productname} 6.6.1, after a content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically scrolls to the previous position.
```

Reviewer: I think it's important to note that we store the existing `scrollTop` value before the content update occurs.
```suggestion
As of {productname} 6.6.1, the extant `scrollTop` value of the iframe dialog component before a content update is stored. After each content update, if the host browser is Firefox or Safari and the previous scroll position was not at the end of the scrollable area, {productname} now programmatically restores the previous scroll position using the stored `scrollTop` value.
```
